js/page-specific/baltcogo-app/BaltCoGoReporterCtrl.js

```diff
@@ -240,30 +240,33 @@
 			});
 		};
 
+		function setupLocation(targetPage) {
+			var isLocationRequired = self.shouldRequireLocation();
+			var isPageTwo = self.page === 2;
+
+			if (isPageTwo && !isLocationRequired) {
+				self.latitude = CONSTANTS.locations.courtHouse.latitude;
+				self.longitude = CONSTANTS.locations.courtHouse.longitude;
+				self.page = targetPage;
+			}
+
+			if (isPageTwo) {
+				$timeout(mapResize, 500);
+			}
+		}
+
 		self.nextClick = function nextClick() {
 			if (validatePanel()) {
 				self.page += 1;
 
-				if (self.page === 2 && self.category.name === 'Website Issue') {
-					self.latitude = CONSTANTS.locations.courtHouse.latitude;
-					self.longitude = CONSTANTS.locations.courtHouse.longitude;
-					self.page = 3;
-				}
-
-				if (self.page === 2) {
-					$timeout(mapResize, 500);
-				}
+				setupLocation(3);
```

Comment:
Did you mean to leave the constant in here? `setupLocation(self.page)` would probably be better.

Reply:
Nevermind. I'm having a morning.